src/code/z_message.c

```diff
@@ -1141,16 +1166,15 @@ void Message_DrawTextWide(PlayState* play, Gfx** gfxP) {
                                                      (msgCtx->msgMode >= MSGMODE_OCARINA_STARTING &&
                                                       msgCtx->msgMode < MSGMODE_SCARECROW_LONG_RECORDING_START))) {
                     j = i;
-                    while (true) {
-                        character = msgCtx->msgBufDecodedWide[j];
-                        if ((character != MESSAGE_WIDE_QUICKTEXT_DISABLE) && (character != MESSAGE_WIDE_PERSISTENT) &&
-                            (character != MESSAGE_WIDE_EVENT) && (character != MESSAGE_WIDE_BOX_BREAK_DELAYED) &&
-                            (character != MESSAGE_WIDE_AWAIT_BUTTON_PRESS) && (character != MESSAGE_WIDE_BOX_BREAK) &&
-                            (character != MESSAGE_WIDE_END)) {
-                            j++;
-                        } else {
+                    for (;;) {
```

Comment:
By any chance can this be rewritten like this instead?
```suggestion
                    for (j = i;; j++) {
```

Reply:
seems to work